mobile/components/views/Attractions/Router.tsx

```diff
@@ -1,31 +1,45 @@
-import { useState } from "react";
-import {DrawerItem} from '@react-navigation/drawer';
-import { View } from "react-native";
+import React, { useState } from 'react';
+import { DrawerItem, DrawerNavigationProp } from '@react-navigation/drawer';
+import { View } from 'react-native';
 
+type DrawerParamList = {
+  Search: any;
+  AddNew: undefined;
```

Comment:
undefined ?

Reply:
I changed it to any. I'm still not familiar with this typing in navigation props.